### PR TITLE
New data frame example

### PR DESCRIPTION
**Pull request by Patowhiz**
Fixes this [comment](https://github.com/africanmathsinitiative/R-Instat/issues/6250#issuecomment-805924953) in issue #6250.

@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

File: instat/dlgNewDataFrame.vb [hunk 1]
```diff
@@ -313,14 +313,17 @@ Public Class dlgNewDataFrame
             lstView.Items.Add(New ListViewItem({"data.frame(x = 1:30, s = rep(""Reading"",30), r = seq(1, 6.8, length=30), t = seq(1, 60, 2))"}))
             lstView.Items.Item(2).ToolTipText = "4 variables, showing use of seq and rep function"
 
-            lstView.Items.Add(New ListViewItem({"data.frame(l = 1:31, d = seq(as.Date(""2013-1-1""), as.Date(""2013-1-31""), ""day""))"}))
-            lstView.Items.Item(3).ToolTipText = "2 variables with dates. Or use Prepare > Column: Date > Generate Dates to add a date variable."
+            lstView.Items.Add(New ListViewItem({"data.frame(date = seq(as.Date(""2021-1-1""), as.Date(""2021-1-31""), ""day""), rain = as.numeric(NA))"}))
+            'lstView.Items.Item(3).ToolTipText = ""
+
+            lstView.Items.Add(New ListViewItem({"data.frame(date = seq(as.Date(""2019-1-1""), as.Date(""2020-12-31""), ""month""), rain = as.numeric(NA), tmin = as.numeric(NA), tmax = as.numeric(NA))"}))
```
Comment (dannyparsons) [inline instat/dlgNewDataFrame.vb:319]: Slightly simplified the element columns. It achieves the same result.
```suggestion
            lstView.Items.Add(New ListViewItem({"data.frame(date = seq(as.Date(""2019-1-1""), as.Date(""2020-12-31""), ""month""), rain = NA_real_, tmin = NA_real_, tmax = NA_real_)"}))

```

**Review by rdstern — CHANGES_REQUESTED** [review]
Both of the new bits work.
a) Include Danny's change to the code for the second example.  It changes from the as.numeric that I suggested.  Keep the as.numeric in the first example - it is quite nice to have the two alternative ways in the 2 examples.
b) Add tooltip:  Two variables with daily dates and rainfall, ready for climatic data entry.  
c) Second tooltip Four variables including monthly dates.  Or use Prepare > Column: Date > Generate Dates to include a date variable.

**Comment by Patowhiz** [comment]
@rdstern @dannyparsons I have made the suggested changes. This is now ready for your review.

**Review by rdstern — APPROVED** [review]
Good

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**